OsmAnd/res/values/strings.xml

```diff
@@ -127,12 +127,12 @@
     <string name="shared_string_cross_buy">Cross-buy</string>
     <string name="map_widget_target_distance">Distance from camera to target</string>
     <string name="ant_widgets">ANT+ widgets</string>
-    <string name="ant_missing_dependency_descr">Could not find the required service \"%1$s\".\nYou need to install or update the ANT+ Plugins Service. \nDo you want to launch the Play Store to do that?</string>
+    <string name="ant_missing_dependency_descr">Could not find the required \"%1$s\" service. \nInstalling the newest version of the ANT+ Plugins service may help. \nOpen the Play Store to get it?"</string>
```

Comment:
"_Installing the newest version of the ANT+ Plugins service may help_" is a rather weird statement, don't you think?  ;)

Reply:
True. And wrong. Reduced it heavily.
Don't know how much double-referencing is going on now, but there was a lot of duplication.